src/components/related/ProductCard.jsx

```diff
@@ -1,31 +1,54 @@
 import React, { useState, useEffect } from 'react';
-import styles from './related.module.css';
+import './related.css';
 import ComparisonModal from './ComparisonModal.jsx';
-import {useProductContext} from '../../context/ProductContext.jsx';
+import {useProductContext, updateProductContext} from '../../context/ProductContext.jsx';
 import imgNotAvailable from '../../../asset/img/image-not-available.png';
 
-const ProductCard = ({relatedProduct}) =>{
-  const { info, styles } = useProductContext();
+const ProductCard = ({relatedProduct, update}) =>{
+  const { info } = useProductContext();
+  // const update = updateProductContext()
+
   const[isOpen, setIsOpen] = useState(false);
 
   const togglePop = () => {
     setIsOpen(!isOpen)
   };
 
-  const salePrice = relatedProduct.styles[0].sale_price
+  const changeCurrentProduct = () =>{
+    update(relatedProduct.info.id)
+    window.scrollTo({
+      top: 0,
+      left: 0,
+      behavior: 'smooth'
+    });
```

Comment:
when clicking the related product img, change context, so the current product will change. and then scroll the page  to overview

Reply:
This scrollTo function is pretty cool! Thanks for the comment!